pkg/server/chaosd/network.go

```diff
@@ -194,13 +209,75 @@ func (s *Server) applyTC(attack *core.NetworkCommand, ipset string, uid string)
 	return nil
 }
 
+func (s *Server) applyEtcHosts(attack *core.NetworkCommand, uid string) error {
+	backupCmd := exec.Command("/bin/bash", "-c", "mv /etc/hosts /etc/hosts.chaosd && touch /etc/hosts")
+	if err := backupCmd.Run(); err != nil {
+		return errors.WithStack(err)
+	}
+
+	fileBytes, err := ioutil.ReadFile("/etc/hosts.chaosd")
+	if err != nil {
+		return errors.WithStack(err)
+	}
+
+	lines := strings.Split(string(fileBytes), "\n")
+
+	needle := "^(\\d{1,3})(\\.\\d{1,3}){3}.*\\b" + attack.DNSHost + "\\b.*"
```

Comment:
how about adding a comment to show an example string that can match this pattern?